theme/package.json

```diff
@@ -8,16 +8,16 @@
     "scripts": {
         "dev": "webpack -d --env dev --config ./webpack.dev.js",
         "dev-watch": "watch 'npm run dev' src",
-        "live": "npm-run-all -p dev-watch proxy:wait",
+        "live": "npm-run-all -p dev-watch proxy browsersync",
         "proxy": "aem-site-theme-builder live",
-        "proxy:wait": "sleep 1 && npm run proxy",
         "prod": "webpack -p --config ./webpack.prod.js",
-        "deploy": "aem-site-theme-builder deploy"
```

Comment:
shouldn't we keep the deploy action?

Reply:
now that we use `npx aem-site-theme-builder deploy` in the Github action, we can remove the deploy step.